src/main/target/SPRACINGF3OSD/target.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+#include "drivers/dma.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+
+    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR, 1 ), // DMA1_Channel2
```

Comment:
They're not used. There's a LOAD of spare solder pads on the PCB, However in my schematic I made careful note to keep these ones free for future use.  If a SPI gyro was added to the schematic it would be possible to fly the board.  As-is, you can fly the board now if you connected up an I2C gyro 😃 Just need a few #ifdefs in the target file...